jekyll/_cci2/circleci-images.md

```diff
@@ -213,14 +217,46 @@ if you want
 to add browsers to the `circleci/golang:1.9` image,
 use the `circleci/golang:1.9-browsers` image.
 
+### Next-Gen Language Images
+{:.no_toc}
+
+Like the legacy images, the next-gen language images are convenience images for common programming languages.
+These images include both the same relevant language and [commonly-used tools](#pre-installed-tools).
+A language image should be listed first under the `docker` key in your configuration,
+making it the [primary container]({{ site.baseurl }}/2.0/glossary/#primary-container){:target="_blank"} during execution.
+
+CircleCI is developing next-gen images for the languages below.
+
+- Android
```

Comment:
I would remove Android for now. The timeline is still a big TBD and may move from a Convenience Image to a machine image. We're still unsure.